.dotnet/src/Custom/Batch/BatchClient.Protocol.cs

```diff
@@ -21,7 +22,7 @@ public partial class BatchClient
     /// <param name="options"> The request options, which can override default behaviors of the client pipeline on a per-call basis. </param>
     /// <exception cref="ArgumentNullException"> <paramref name="content"/> is null. </exception>
     /// <exception cref="ClientResultException"> Service returned a non-success status code. </exception>
-    /// <returns> A <see cref="CreateBatchOperation"/> that can be used to wait for 
+    /// <returns> A <see cref="CreateCreateBatchOperation"/> that can be used to wait for 
```

Comment:
Is the name in the cref correct?

Reply:
I thought I had hit all of these. Good catch!